db/migrate/20220324101145_create_comments.rb

```diff
@@ -0,0 +1,8 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] <b>REQUIRED:</b> Please create the corresponding model for comment and implement relationship between `Comment` and other models following the this [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)